core/test-loader.ts

```diff
@@ -1,6 +1,7 @@
 import { ITestFixture, ITest, ITestCase } from "./_interfaces";
 import { FileRequirer } from "./_core";
 import { TestFixture, METADATA_KEYS } from "./alsatian-core";
+const getFunctionName = require("fn-name");
```

Comment:
Looking at the libraries source code I don't think it'll we'll have a use case for it since function.name is becoming standardised and for all our currently supported environments it behaves as expected. The only use case I can think of is for legacy browser support (IE) which the vast majority of developers don't use so if we were to make available a browser version then it'd probably not target IE so we have no need for a fallover

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Function/name